pkg/manager-server/file_system_api.go

```diff
@@ -212,6 +212,15 @@ type FileSystemOemLvm struct {
 
 	// The lvcreate commandline, minus the "lvcreate" command.
 	LvCreate string `json:"LvCreate,omitempty"`
+
+	// The vgchange commandline, minus the "vgchange" command.
```

Comment:
Minus the "--activate" arg as well.

Vgchange has two cases, activate and deactivate, so this gets crummy because they ought to be separate fields here.

Reply:
Actually, 3 options if you include --lock-start. But I didn't see any reason to break out vgchange activate "y" or "n" and I didn't want to dirty things up (anymore than they are). It gets even worse if you see GFS2 use `vgchange --activate ys $VG_NAME` (Note the 's')

I suppose I could nest things
```
VgChange struct {
  Activate string   // defaults to "--activate y $VG_NAME" for XFS/raw, "--activate ys $VG_NAME" for GFS2
  Deactivate string // defaults to "--activate n $VG_NAME"
  LockStart string  // defaults to "--lock-start $VG_NAME"
}